lib/libzrepl/data_conn.c

```diff
@@ -1423,6 +1484,25 @@ uzfs_zvol_io_ack_sender(void *arg)
 			continue;
 		}
 
+		if (zio_cmd->hdr.opcode == ZVOL_OPCODE_REBUILD_SNAP_DONE) {
+			rc = uzfs_zvol_socket_write(zio_cmd->conn,
+			    (char *)&zio_cmd->buf, sizeof (zio_cmd->hdr.len));
+			if (rc == -1) {
```

Comment:
lets do a 'goto' call rather than duplicate code or some code re-org would be better